infra/base-images/base-builder/Dockerfile

```diff
@@ -125,8 +125,7 @@ RUN cd $SRC && \
 
 RUN cd $SRC && \
     git clone --depth 1 https://github.com/google/centipede.git && \
-    cd centipede && \
-    sed -i.bck '$s/$/ --cxxopt=-stdlib=libc++ --linkopt=-lc++/' .bazelrc
+    echo 'build --cxxopt=-stdlib=libc++ --linkopt=-lc++' >> centipede/.bazelrc
```

Comment:
Does this still work if we write to $HOME/.bazelrc? Will this automatically get concatenated in the final build command?